src/palace/manager/api/admin/controller/collection_settings.py

```diff
@@ -142,6 +142,17 @@ def process_post(self) -> Response | ProblemDetail:
             # Trigger a site configuration change
             site_configuration_has_changed(self._db)
 
+            # If we have an importer task for this protocol, we start it
+            # in the background, so that the collection is ready to go
+            # as quickly as possible.
+            try:
+                impl_cls.import_task(integration.collection.id).apply_async(
+                    countdown=10  # Delay the task to ensure the collection is created the task starts
```

Comment:
Do we want the import task to be run on every change (i.e., some changes -- e.g., name change -- may not require an import)? Although, I suppose that if we were to be selective, that selection probably needs to happen in the implementation class, since difference settings would require import in different implementations. Maybe food for thought for a future optimization, if it were to prove useful.